src/lib/components/table/QTable.svelte

```diff
@@ -2,45 +2,45 @@
   import { QIcon, QSelect, QBtn } from "$lib";
   import type { QTableProps, QTableColumn, QTableRow, QTableSort } from "./props";
 
-  export let columns: QTableProps["columns"] = [],
-    rows: QTableProps["rows"] = [],
-    flat: QTableProps["flat"] = undefined,
-    bordered: QTableProps["bordered"] = undefined,
-    dense: QTableProps["dense"] = false,
-    userClasses: QTableProps["userClasses"] = "";
-  export { userClasses as class };
+  let {
+    columns = [],
+    rows = [],
+    flat = undefined,
+    bordered = undefined,
+    dense = false,
+    bodyCell = undefined,
```

Comment:
You don't need `flat = undefined`, `flat` is enough

Reply:
Yeah we have many of these unnecessary default values, also Docs sometimes differ from the actual value. We should discuss a general approach.